src/email_relay/conf.py

```diff
@@ -19,6 +19,10 @@ class AppSettings:
     EMAIL_THROTTLE: int = 0
     MESSAGES_BATCH_SIZE: int | None = None
     MESSAGES_RETENTION_SECONDS: int | None = None
+    RELAY_HEALTHCHECK_METHOD: str = "GET"
+    RELAY_HEALTHCHECK_STATUS_CODE: int = 200
+    RELAY_HEALTHCHECK_TIMEOUT: float = 5.0
```

Comment:
Depending on how fancy you want to get with this, timeout can be either a `float` or `tuple(float, float)` -> https://requests.readthedocs.io/en/latest/user/advanced/#timeouts 

This shouldn't hold up the PR, but I suspect you will get someone ask about it down the road.